src/commands/alias.js

```diff
@@ -64,9 +62,8 @@ const alias = {
     } else {
       if (value) {
         vorpal.localStorage.setItem(`alias|${key}`, value);
-        _.remove(all, function (val) {
-          return val === key;
-        });
+        const index = all.indexOf(key);
```

Comment:
Whoops @blockloop 

On future PRs, make sure you run `gulp` first to build (sorry didn't catch that). This broke alias:

```js
const index = all.indexOf(key);
all.splice(index, 1);
```